ui/messages/ArbitrumL2Messages.tsx

```diff
@@ -0,0 +1,102 @@
+import { Hide, Show, Skeleton } from '@chakra-ui/react';
```

Comment:
To keep the folder structure consistent with `/deposits`, `/withdrawals`, and others, I suggest placing the component in the `/arbitrum` sub-folder.

Reply:
we don't have other types with messages, so I think it's unnecessary here